api-documentation.js

```diff
@@ -325,6 +383,11 @@ class ApiDocumentation extends AmfHelperMixin(LitElement) {
     }
     this._selected = value;
     this.__amfChanged();
+
+    if (!this.handleNavigationEvents) {
```

Comment:
Just a question, what is that? And why is that? :)

Reply:
As I understand it, this boolean that already existed in the component enables it to start listening to the `api-navigation-changed` event. Since when navigation occurs in the case this in true, both properties are changed at the same time, I prefer for `this._updateServers()` to run only once. What I thought back then was that if we are already updating servers when navigation changes, we don't need to do it when individual properties change. But now that you have pointed this out, I realize that the component could both have `handleNavigationEvents` and require to change `selected` and `selectedType` individually. I'll delete this change.